src/index.js

```diff
@@ -7,6 +7,10 @@ import './bots/comp';
 import './bots/ping';
 import './bots/pins';
 
+/**
+ * Connect to MongoDB ip/port from .env
+ * @param {string} MONGODB - the ip/port for the MongoDB connection.
+ */
 mongoose
```

Comment:
Same as my other comment above. Since this isn't a declaration or export it isn't really a docstring.